src/OrchardCore.Modules/OrchardCore.AdminDashboard/Migrations.cs

```diff
@@ -0,0 +1,43 @@
+using OrchardCore.Data.Migration;
+using OrchardCore.AdminDashboard.Indexes;
+using YesSql.Sql;
+using OrchardCore.ContentManagement.Metadata;
+using OrchardCore.ContentManagement.Metadata.Settings;
+using OrchardCore.Recipes.Services;
+using System.Threading.Tasks;
+
+namespace OrchardCore.AdminDashboard
+{
+    public class Migrations : DataMigration
+    {
+        private IContentDefinitionManager _contentDefinitionManager;
+        private readonly IRecipeMigrator _recipeMigrator;
+
+        public Migrations(IContentDefinitionManager contentDefinitionManager, IRecipeMigrator recipeMigrator)
+        {
+            _contentDefinitionManager = contentDefinitionManager;
+            _recipeMigrator = recipeMigrator;
+        }
+
+        public int Create()
+        {
+            SchemaBuilder.CreateMapIndexTable<DashboardPartIndex>(table => table
+               .Column<double>("Position")
+            );
+
+            _contentDefinitionManager.AlterPartDefinition("DashboardPart", builder => builder
+                .Attachable()
+                .WithDescription("Provides a way to add widgets to a dashboard.")
+                );
```

Comment:
How about Weld `DashboardPart` in Driver, instead of defining in migraion?

Any content item created with stereotype Dashboard item should automatically weld to this part, Like we have as in FlowMetatadata and WidgetMetadata

Reply:
We talked about it, but I think the reason is that there was not much advantage to weld it instead of doing it on purpose, either by code or UI. It's just a requirement. Welding makes things too magical (imo).

Reply:
We are only hoping that you can take this feature and make a nice editor when it's merged.

Reply:
Only reason, I was thinking on to welding automatically to `Dashboard` stereotype is because, it's should common behavior of all Dashboard content item (more like metadata to dashboard functionality than manageable content part), and shouldn't require user to explicitly define part on every content type definition only defining stereotype `Dashboard ` should be enough.

Also editing Position( and Width, Height for resizing) property is done on manage page (when drag drop is implemented) and  editing these meta data properties on edit content screen becomes less significant.